src/amazonq/webview/ui/main.ts

```diff
@@ -77,7 +87,7 @@ export const createMynahUI = (ideApi: any, featureDevInitEnabled: boolean, gumby
             }
 
             // Unlock every authenticated tab that is now authenticated
-            if (featureDevEnabled) {
+            if (idcEnabled) {
```

Comment:
Why is this changing from if(featureDevEnabled)?
We said earlier that we wanted the IDE to be as ignorant as possible about *why* the feature was visible or not.

Reply:
I can change the name to amazonQConnected if that makes it more clear? Alternatively I could have changed `onUpdateAuthentication` to take in another field for gumbyEnabled but that doesn't really make sense since featureDev and gumby are enabled in all the same cases (and makes that part of the code harder to maintain)